core/src/storage/__init__.py

```diff
@@ -20,6 +16,14 @@ def init_unlocked() -> None:
     version = device.get_version()
     if version == common.STORAGE_VERSION_01:
         _migrate_from_version_01()
+    if device.is_version_stored() and not device.is_initialized():
+        common.set_bool(common.APP_DEVICE, device.INITIALIZED, True, public=True)
+
+
+def reset() -> None:
```

Comment:
please add a docstring with a brief explanation of what's going on here

Reply:
Fixed.